app/Providers/AppServiceProvider.php

```diff
@@ -14,7 +19,15 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        $this->app->bind(RequestLoggerInterface::class, function (){
+            $logger = NULL;
+            if(App::environment('local')){
```

Comment:
you can't allow to return null here. Simple use production environment and on else use FullRequestLogger